R/viz.R

```diff
@@ -2608,9 +2608,10 @@ theme_Morons = function(base_size = 14,
 #' Create a forest plot comparing mutation frequencies for a set of genes between two groups.
 #'
 #' @param maf A maf data frame. Minimum required columns are Hugo_Symbol and Tumor_Sample_Barcode.
-#' @param mutmat Optional argument for binary mutation matrix. If not supplied, function will generate this amtrix from the file used in argument "maf".
+#' @param mutmat Optional argument for binary mutation matrix. If not supplied, function will generate this matrix from the file used in argument "maf".
 #' @param metadata Metadata for the comparisons. Minimum required columns are Tumor_Sample_Barcode and the column assigning each case to one of two groups.
-#' @param genes An optional list of genes to restrict your plot to.
+#' @param genes An optional list of genes to restrict your plot to. If no gene-list is supplied, the function will extract all mutated genes from the incoming maf. See n_mutations parameter for more info.
+#' @param n_mutations Optional parameter for when no gene list is provided. This parameter ensures only genes with n mutations are kept in the gene list. Default value is 1, this means all genes in the incoming maf will be plotted.
```

Comment:
How about min_mutations so it's more self-explanatory?

Reply:
I like it, thanks!